clairmeta/dcp_check_subtitle.py

```diff
@@ -66,9 +67,7 @@ def ticks_to_frame(self, tick, edit_rate):
         tick = int(tick)
         time_base = 1.0 / edit_rate
 
-        # Ceiling division. Ugly, but avoids importing math.ceil
-        # https://stackoverflow.com/questions/14822184/is-there-a-ceiling-equivalent-of-operator-in-python#17511341
-        return int(-(-(tick * 0.004) // time_base))
+        return int((tick * 0.004) // time_base)
```

Comment:
Must admit I don't remember the exact reasoning behind this.

Did you check it's safe to remove that ceiling division ?

Reply:
My reasoning when first implementing this whas to round the calculated fractional frame to the nearest whole frame number (up or down). When thinking more about this, it does not make sense to round up, because then a subtitle is to be displayed later than actually specified (albeit only by a small fraction of frames).

I've had a couple of false warnings with the rounding up (ceiling division), which disappeared with this patch.

Reply:
Ok, I trust your judgment on this one.

Did some digging and found this was added from here https://github.com/Ymagis/ClairMeta/pull/58

My guess is that we conclude the DCP you mentioned back then was invalid, or at least the subtitle is invisible ? Wonder if the projector actually showed it or not, would probably require frame by frame capability to check.